src/Request/RequestCollectionFactory.php

```diff
@@ -30,7 +30,7 @@ public function create(string $rawRequest): RequestCollection
 		if (is_array($requestData)) {
 			foreach ($requestData as $oneRequestData) {
 				$collection->attach(
-					$this->createRequestFromRequestData($oneRequestData)
+					$this->createRequestFromRequestData($oneRequestData) // @phpstan-ignore argument.type
```

Comment:
Phpstan is here to catch edge cases before there is a runtime error. We shouldn't really just ignore everything it throws at us.